docs: edit custom launch template updates for eks (#1118)
rancher/rancher-docs: docs/reference-guides/cluster-configuration/rancher-server-configuration/eks-cluster-configuration.md

| User Data | Cloud init script in [MIME multi-part format](https://docs.aws.amazon.com/eks/latest/userguide/launch-templates.html#launch-template-user-data) | Optional |
| Instance Resource Tags | Tag each EC2 instance and its volumes in the node group | Optional |

:::caution

> Node groups created from a custom launch template cannot be directly updated to a newer Kubernetes version. The sole method to update these node groups is by creating a new version of the launch template and associating this node group with the updated version.

Reviewer: ```suggestion
You can't directly update a node group to a newer Kubernetes version if the node group was created from a custom launch template. You must create a new launch template with the proper Kubernetes version, and associate the node group with the new template.
```

Revising for style guide and voice.
